### DISCUSSION TITLE
Active - Backup Cluster

### DISCUSSION THREAD

**Discussion by wyphorn-png** [patroni/patroni#3091] [Q&A] [answered]
Hi,

I have a cluster of two servers. Now I want to set node1 as primary server and node2 as backup server. So node2 should be replicator as long as node1 is there. In case of node1 is down, node2 should go primary but should directly switch to replicator when node1 returns. How can I set this up?

**Comment by CyberDem0n** [comment, marked as answer]
Patroni is trying to minimize the number of failovers/switchovers.
In case if the dead node (former primary) is returned, it will always join as a standby.
You can always switchover to it manually.
